README.md

```diff
@@ -93,6 +93,16 @@ Go to Dashboard > [API Keys](https://platform.openai.com/api-keys) in an OpenAI
 
 Create a Gemini [API Key](https://aistudio.google.com/app/apikey?_gl=1*nc11k*_ga*OTIzNzIyMjM0LjE3MzYzNjM5Nzc.*_ga_P1DBVKWT6V*MTczNjM2Mzk3Ni4xLjAuMTczNjM2Mzk3Ni42MC4wLjEyOTEyNzA3OTM.) in Google AI Studio. Billing can also be set up within Studio however many of the services are free of charge until specific rate limits are met. See pricing information [here](https://ai.google.dev/pricing?_gl=1*1ktr08b*_ga*OTIzNzIyMjM0LjE3MzYzNjM5Nzc.*_ga_P1DBVKWT6V*MTczNjM2Mzk3Ni4xLjEuMTczNjM2Mzk3OS41Ny4wLjEyOTEyNzA3OTM.#1_5flash).
 
+## Analyzing differences between US Core versions
+
+The `us_core_differences` folder contains scripts to detect differences in the IG narrative between versions of the US Core IG. The `process_igs.py` script will download two versions of the IG, extract the narrative from html files, and convert it to markdown. The `compare_igs.py` script will identify differences in the narrative between the two versions. The differences will be outputted to `ARTIFACTS_DIRECTORY/ig/differences.md`.
```

Comment:
If the artifacts-directory is not specified, it outputs the differences to `demo-artifacts/ig/differences.md` (I think?) so maybe change the language here to say that or like `<input artifacts_dir>/ig/differences.md`?